manifests/osfamily/redhat.pp

```diff
@@ -20,7 +20,6 @@
       'Amazon': {
         $major_version = $facts['os']['release']['major']
         $amz_el_version = "${major_version}" ? {
-          '2'              => '7',
```

Comment:
Removing this is correct for amazon-2-aarch64, but breaks amazon-2-x86_64 (since we need to install RHEL 7 packages.